### DISCUSSION TITLE
Full size graph

### DISCUSSION THREAD

**Discussion by anoril1979** [kalkih/mini-graph-card#795] [Q&A]
Hello,

First: many thanks for that card which is a gold nugget!

My question: is it possible to get the graph to fill the card, with text over it.
My main concern is that I got a lot span I don't know how to remove... (if one has an answer for this first, I would be quite happy)
![image](https://user-images.githubusercontent.com/12977446/171031289-82a9074f-ef72-4bba-a657-f12c1f64932a.png)

Best regards,
Paul

**Comment by ildar170975** [comment]
Try playing with card-mod

**Reply by anoril1979** [reply]
I played with it. A very good plugin.
But I'm not good enough at CSS to get the point. Too many hours for nothing: got all the containers the right place and size BUT graphs are still small (500×100). :/

**Reply by ildar170975** [reply]
> graphs are still small (500×100)

Increase graphs then , there is a stock `height` property

**Reply by anoril1979** [reply]
Hi! Can't get this working. Height is already 100%. If I set it to 200% or even 200px, the box is well set, but the content (lines, dots, etc.) is still 500×100... When I force the viewport (0 0 500 100) to something else: no visible change. I don't really want to modify this directly into the plugin code (as I don't want all the graphs to be changed :P )

**Reply by rotweiss4** [reply]
Hi, found a solution? im looking exactly this.

**Reply by anoril1979** [reply]
Nope, not yet. Did not take time to learn CSS and n otime either to dig the existing code. :/

**Reply by Head** [reply]
Try something like this:
```
card_mod:
  style: |
    ha-card {
      padding: 0 !important;
    }
    .header {
      padding-top: 10px;
      padding-bottom: 0px;
      z-index: 1;
      background: rgba(255, 255, 255, 0.5);
    }
    .states {
      padding-bottom: 20px;
      z-index: 1;
      text-shadow: white 2px 0 2px;
      background: rgba(255, 255, 255, 0.5);
    }
    .graph {
      position: absolute !important;
      top: 0px;
      height: 100%;
      padding-top: 0px !important;
    }
```
